source/configure/database-configuration-settings.rst

```diff
@@ -388,27 +388,111 @@ Replica lag settings
+| **Notes**:                                                                                                                                |
 |                                                                                                                                           |
-| For AWS Aurora instances, ``QueryAbsoluteLag`` can be:                                                                                    |
-|                                                                                                                                           |
-| .. code-block:: sql                                                                                                                       |
-|                                                                                                                                           |
-|   select server_id, highest_lsn_rcvd-durable_lsn as bindiff from aurora_global_db_instance_status() where server_id=<>                    |
-|                                                                                                                                           |
-| And for AWS Aurora instances, ``QueryTimeLag`` can be:                                                                                    |
-|                                                                                                                                           |
-| .. code-block:: sql                                                                                                                       |
-|                                                                                                                                           |
-|   select server_id, visibility_lag_in_msec from aurora_global_db_instance_status() where server_id=<>                                     |
-|                                                                                                                                           |
-| For MySQL Group Replication, the absolute lag can be measured from the number of pending transactions in the applier queue:               |
-|                                                                                                                                           |
-| .. code-block:: sql                                                                                                                       |
-|                                                                                                                                           |
-|   select member_id, count_transactions_remote_in_applier_queue FROM performance_schema.replication_group_member_stats where member_id=<>  |
+| - The ``QueryAbsoluteLag`` and ``QueryTimeLag`` queries must return a single row.                                                         |
+| - To properly monitor this you must setup `performance monitoring </scale/performance-monitoring.html>`__ for Mattermost.    |
```

Comment:
FYI: There's a pipe misaligned on this table that will prevent the table from displaying.